src/Education/IRuleHelpXamlBuilder.cs

```diff
@@ -0,0 +1,58 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System.IO;
+using System.Windows.Documents;
+using System.Windows.Markup;
+using System.Xml;
+using SonarLint.VisualStudio.Rules;
+
+namespace SonarLint.VisualStudio.Education
+{
+    internal interface IRuleHelpXamlBuilder
+    {
+        FlowDocument CreateFromHTML(IRuleHelp ruleHelp);
+    }
+
+    internal class RuleHelpXamlBuilder : IRuleHelpXamlBuilder
+    {
+        public FlowDocument CreateFromHTML(IRuleHelp ruleHelp)
+        {
+            // TODO: Convert HTML to FlowChart
+
+            var placeholder = $@"
+                <FlowDocument xml:space=""preserve"" xmlns=""http://schemas.microsoft.com/winfx/2006/xaml/presentation"">
+                    <Paragraph>Rule: {ruleHelp.RuleKey}</Paragraph>
+                    <Paragraph>Language: {ruleHelp.Language}</Paragraph>
```

Comment:
Should perhaps be `ruleHelp.Language.Name` rather than just `ruleHelp.Language`.

`Language` is a class in `Core` with multiple properties, and it doesn't override `ToString` so the default text probably isn't what we want.